dwave/plugins/sklearn/transformers.py

```diff
@@ -0,0 +1,257 @@
+from sklearn.base import BaseEstimator
+from sklearn.feature_selection import SelectorMixin
+from dwave.system import LeapHybridCQMSampler
+import dimod
+import numpy as np
+import pandas as pd
+import tempfile
+
+import logging
+
+class HybridFeatureSelection(BaseEstimator, SelectorMixin):
```

Comment:
I am not sold on the name. Ideally it would be more descriptive. [Most of the others](https://github.com/scikit-learn/scikit-learn/blob/20ad9cd62cbef744eb28b29ab8a33fc3b51f8a3e/sklearn/feature_selection/__init__.py#L34-L52) are imperative like `SelectKBest`.